osbs/conf.py

```diff
@@ -532,3 +532,27 @@ def get_equal_labels(self):
                 raise OsbsValidationException("Wrong equal_labels configuration")
 
         return equal_labels
+
+    def get_node_selector_platform(self, platforms):
```

Comment:
Can we have a docstring explaining why this takes a parameter i.e. it returns the node selector for the given platform. Also, it should take a `platform` (singular).